sdk/test-utils/perfstress/package.json

```diff
@@ -81,6 +82,7 @@
     "karma-env-preprocessor": "^0.1.1",
     "prettier": "^1.16.4",
     "rimraf": "^3.0.0",
-    "typescript": "4.1.2"
+    "typescript": "4.1.2",
+    "typedoc": "0.15.0"
```

Comment:
I don't believe this is a published package, so we can skip this. Also add `private: true` here. @HarshaNalluru to confirm

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/12941/commits/68f3a198a41640f0e47b4ffa4917a0e46291c89b has addressed most of my feedback, only remaining thing is the test-utils-recorder package

Reply:
@ramya-rao-a done.

Reply:
Not a published package, can be skipped

Reply:
Also, both test-utils packages already have `private: true` in their `package.json`.